pkg/assembler/graphql/schema/hasSBOM.graphql

```diff
@@ -69,4 +69,6 @@ extend type Query {
 extend type Mutation {
   "Certifies that a package or artifact has an SBOM."
   ingestHasSBOM(subject: PackageOrArtifactInput!, hasSBOM: HasSBOMInputSpec!): HasSBOM!
+  "Bulk ingest that package or artifact has an SBOM."
+  ingestHasSBOMs(subjects: PackageOrArtifactInputs!, hasSBOMs: [HasSBOMInputSpec!]!): [HasSBOM!]!
```

Comment:
Shouldn't this be a list of `PackageOrArtifactInput` instead?

Reply:
so that does not work on the client side as you can only specify a package for the subject:

```
operations/hasSBOM.graphql:35: query-spec does not match schema: Field "packages" is not defined by type "PackageOrArtifactInput". Did you mean "package"?
```

Had to create a special input:
```
input PackageOrArtifactInputs {
  packages: [PkgInputSpec!]
  artifacts: [ArtifactInputSpec!]
}
```
so you can pass in a list of packages or artifacts

Reply:
Was running into this error:

```
operations/hasSBOM.graphql:35: query-spec does not match schema: Variable "$pkgs" of type "[PkgInputSpec!]!" used in position expecting type "[PackageOrArtifactInput!]!".
```
that was the reason to create the above.

Reply:
Oh, it's the client constraint